### ISSUE TITLE
training on cash rather than BTC.

### ISSUE THREAD

**Issue by AhmMontasser** [ZhengyaoJiang/PGPortfolio#56]
Hello. 

I've been trying to train the model on cash and assets rather than BTC and cryptos.

I change initial cash everywhere i know (trader classes) and did the required changes to fetch assets data and return global panel and data management is very good.

However, when i start training. The agent seems not recognizing the cash change, it's still outputting

portfolio value on test set is 1.412 for example.  

what did i miss ?

**Comment by ZhengyaoJiang** [comment]
Hi,
What do you exactly mean by "train on cash and assets"?
I think the BTC is just cash and alter-coins are assets in this project?

Regards
Zhengyao

**Comment by AhmMontasser** [comment]
I mean to train the model on training in stock market. 

I edited globaldatamatrix to fetch stocks data instead of cryptos and changed the initial_btc at the trader class to 1000000 for example. 

when it's training it doesn't raise an error but doesn't recognize the change in initial value.

**Comment by ZhengyaoJiang** [comment]
In pre-training stage, the initial portfolio value is always one.
Only in backtest stage, the initial_btc will affect the final results.

> portfolio value on test set is 1.412 for example.

This seems to indicate the migration from crypto-market to stock market has been successful since there is 40% return.

**Comment by AhmMontasser** [comment]
Great, thank you

**Comment by AhmMontasser** [comment]
Hello, 

When i did backtesting on real stock data, i noticed that the agent decides to invest all capital on one stock for most of the time and completely ignores other assets.
for example, the raw omega for number of backtesting steps on Daw Jones data is similar to this.

the step is 397
the raw omega is [  0.00000000e+00   0.00000000e+00   0.00000000e+00   0.00000000e+00
   0.00000000e+00   9.99996185e-01   0.00000000e+00   0.00000000e+00
   3.82340568e-06   0.00000000e+00   0.00000000e+00]

the step is 429
the raw omega is [  0.00000000e+00   0.00000000e+00   0.00000000e+00   9.99953985e-01
   0.00000000e+00   4.60227748e-05   0.00000000e+00   0.00000000e+00
   0.00000000e+00   3.16850148e-23   0.00000000e+00]

The step is 472
the raw omega is [  0.00000000e+00   0.00000000e+00   1.53690362e-02   0.00000000e+00
   0.00000000e+00   4.71898817e-20   0.00000000e+00   0.00000000e+00
   0.00000000e+00   0.00000000e+00   9.84631002e-01]

the step is 524
the raw omega is [  0.00000000e+00   0.00000000e+00   9.99999881e-01   0.00000000e+00
   0.00000000e+00   8.38515799e-18   0.00000000e+00   1.64659710e-07
   0.00000000e+00   0.00000000e+00   0.00000000e+00]

the step is 545
the raw omega is [ 0.  0.  0.  0.  0.  1.  0.  0.  0.  0.  0.]

What do you think could be causing this ? Is it logical to have the Agent voting for only 1 asset every time ?
